### PR TITLE
[222] Sign-Up Form Validation - Mobile

### PR DESCRIPTION
**Pull request by HalimahOladimeji**
Screenshot 2021-01-10 200521

### CODE REVIEW

File: submissions/HalimahOladimeji/HTML-forms/contactus.html [hunk 1]
```diff
@@ -1,88 +0,0 @@
-<!DOCTYPE html>
```
Comment (adaugochi) [inline submissions/HalimahOladimeji/HTML-forms/contactus.html:1]: Is there a reason why you are deleting this file

File: submissions/HalimahOladimeji/CSS/contactus.css [hunk 2]
```diff
@@ -1,228 +0,0 @@
-body {
```
Comment (adaugochi) [inline submissions/HalimahOladimeji/CSS/contactus.css:1]: Is there a reason why you are deleting this file

**Review by adaugochi — CHANGES_REQUESTED** [review]
Well done. 

The function `firstNameFieldValidation, lastNameFieldValidation` etc literally does the same thing. Why not create a single function for that and pass the things that make them different as a parameter.

Also, I thing you should check if the input field is empty first before checking for if it matches the regex

**Comment by HalimahOladimeji** [comment]
I deleted the contact us files because it has been validated and i intend to upload it in my next PR. I hope that is okay.

**Comment by adaugochi** [comment]
> I deleted the contact us files because it has been validated and I intend to upload it in my next PR. I hope that is okay.

it is not that okay. Even if you intend to change it, you should have waited still then. I will advise you revert the changes back.

**Review by adaugochi — APPROVED** [review]
Well done

**Pull request merged**